contracts-periphery/script/Deploy.s.sol

```diff
@@ -1,15 +1,35 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.16;
 
-import "forge-std/Script.sol";
-import {UmbraBatchSend} from "src/UmbraBatchSend.sol";
-import {IUmbra} from "src/interface/IUmbra.sol";
+/// @title Deploy a contract to multiple networks.
+/// @notice Run the script with `forge script Deploy --private-key <privateKey>`
+/// and add `--broadcast` to broadcast the tx.
+/// @dev The contract that's being deployed is specified in `BaseScript.sol`.
+/// Specify additional netowrk aliases in [rpc_enpoints] in `forge.toml`
+/// before adding them to the list of networks below.
+/// Add corresponding `env` variables in a `.env` file, such as ETH_RPC_URL=<URL>
```

Comment:
A file can have multiple contracts, so this should be directly above the `contract` line. Since this script is for devs only we can just do this (note some typo fixes):
```suggestion
/// @dev Deploys the UmbraBatchSend contract to all supported networks networks, skipping
/// networks that already have the contract
/// @dev Run the script with `forge script Deploy --private-key <privateKey>`
/// and add `--broadcast` to broadcast the tx.
/// @dev Specify additional network aliases in [rpc_endpoints] in `forge.toml`
/// before adding them to the list of networks below, and add corresponding environment
/// variables to the `.env` and `.env.template` files.
```

Reply:
Makes sense, thanks! https://github.com/ScopeLift/umbra-protocol/pull/504/commits/663b1c2987ea5cb67ed057e61fff71d69ebf60bd